packages/persist/lib/server.ts

```diff
@@ -39,7 +39,7 @@ server.post(
             msg: z.string()
         })
     }),
-    persistController.saveActivityLog
+    persistController.saveActivityLog.bind(persistController)
```

Comment:
why is bind necessary here?

Reply:
it's not necessary to make it work but because it's in a class `this` reference can be mistakenly overrode.
overall our usage of class is probably unnecessary

Reply:
what would you recommend instead of using a class?

Reply:
straight up function, if we had state it would be debatable but it's not the case and I would argue keeping state between api calls is dangerous